core/tools/blueos_startup_update/blueos_startup_update.py

```diff
@@ -263,6 +263,29 @@ def update_i2c4_symlink() -> bool:
     return False  # This patch doesn't require restart to take effect
 
 
+def revert_update_dwc2() -> bool:
+    """
+    Removes dwc2 configuration from cmdline.txt
+    This was being wrongly applied on Pi3 due to a bad host_cpu check.
+    """
+
+    # Remove dwc2 module configuration from cmdline
+    unpatched_cmdline_content = load_file(cmdline_file).replace("\n", "").split(" ")
+    cmdline_content = []
+    for item in unpatched_cmdline_content:
+        if "dwc2" not in item and "g_ether" not in item:
+            cmdline_content.append(item)
+
+    # Save if needed, with backup
+    if unpatched_cmdline_content == cmdline_content:
+        return False
+    save_file(cmdline_file, " ".join(cmdline_content), "before_revert_update_dwc2")
+
+    # Patch applied and system needs to be restarted for it to take effect
+    return True
+    return True
```

Comment:
oh it seems a rebase may have gone wrong and there are two `return True` on this commit.
not a blocker for me though 🤷‍♂️

Reply:
hahaha yeah, that was there, ty!